src/src/camp/camp.core.service.ts

```diff
@@ -289,4 +289,26 @@ export class CampCoreService {
 
 		return response;
 	}
+
+	public async updateCampStatus(
```

Comment:
change `updateCampStatus` to `updateCampGroup`

Reply:
resolved